src/CreatorTokenSwapRouter.sol

```diff
@@ -30,6 +37,15 @@ contract CreatorTokenSwapRouter {
   // Uniswap's `QuoterV2` contract off-chain.
   // Check Uniswap docs: https://docs.uniswap.org/contracts/v3/reference/periphery/lens/QuoterV2 and
   // `quote` function in `test/SwapRouter.fork.8453.t.sol.sol`
+  /// @notice Buys a Creator Token with ETH from the caller.
+  /// @dev Requires an ETH payment.
+  /// @dev You can get a quote for the amount of ETH you have to pay to buy a creator token by
+  /// calling Uniswap's `QuoterV2` contract off-chain. Check Uniswap docs:
+  /// https://docs.uniswap.org/contracts/v3/reference/periphery/lens/QuoterV2 and `quote` function
+  /// in `test/SwapRouter.fork.8453.t.sol.sol`
+  /// @param _creatorToken Address of the Creator Token contract.
+  /// @param _maxPayment Maximum amount of USDC willing to be paid for the token.
+  /// @return _amountOut The amount of USDC paid.
   function buyWithEth(address _creatorToken, uint256 _maxPayment)
```

Comment:
This and other buy methods should have a note that you can/should send the ETH value needed for the purchase with the transaction.

Reply:
Added them here: [e159a53](https://github.com/showtime-xyz/creator-tokens/pull/46/commits/e159a53031291a7ef77626ca2d681aec08053372)

Reply:
Sorry for the additional nit here, but I think we want to be clear that the ETH:

1) Must be sent as `value` with the transaction and
2) Will be swapped for amount of USDC required to buy the creator token(s)

Can we adjust the wording a bit to make those two things crystal clear? Just want to be extra explicit and technical since this is a pretty sensitive thing.

Reply:
How's this https://github.com/showtime-xyz/creator-tokens/pull/46/commits/6ae0bc7e86612507703a56a5cd487ab4ca9bc40e